src/Neon/Encoder.php

```diff
@@ -28,6 +28,13 @@ public function encode($var, $options = NULL)
 			return $var->format('Y-m-d H:i:s O');
 
 		} elseif ($var instanceof Entity) {
+			if($var->value === Neon::CHAIN) {
+				$s = '';
+				foreach($var->attributes as $attr) {
+					$s .= $this->encode($attr);
+				}
+				return $s;
```

Comment:
Or this

~~~php
return array_reduce($var->attributes, function ($carry, $item) {
    return $carry . $this->encode($item);
});
~~~